packages/actor-query-operation-group/lib/ActorQueryOperationGroup.ts

```diff
@@ -0,0 +1,84 @@
+
+import { ArrayIterator } from 'asynciterator';
+import { termToString } from 'rdf-string';
+import { Algebra } from "sparqlalgebrajs";
+import { SimpleEvaluator } from 'sparqlee';
+
+import {
+  ActorQueryOperation,
+  ActorQueryOperationTypedMediated,
+  IActorQueryOperationOutputBindings,
+  IActorQueryOperationTypedMediatedArgs,
+} from "@comunica/bus-query-operation";
+import { ActionContext, IActorTest } from "@comunica/core";
+
+import { GroupsState } from './GroupsState';
+
+/**
+ * A comunica Group Query Operation Actor.
+ */
+export class ActorQueryOperationGroup extends ActorQueryOperationTypedMediated<Algebra.Group> {
+
+  constructor(args: IActorQueryOperationTypedMediatedArgs) {
+    super(args, 'group');
+  }
+
+  public async testOperation(pattern: Algebra.Group, context: ActionContext): Promise<IActorTest> {
+    for (const i in pattern.aggregates) {
+      // Will throw for unsupported expressions
+      const _ = new SimpleEvaluator(pattern.aggregates[i].expression);
```

Comment:
```suggestion
      new SimpleEvaluator(pattern.aggregates[i].expression);
```


Unless the linter complains about this?

Reply:
The linter complains indeed.